packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.types.ts

```diff
@@ -46,7 +47,27 @@ export interface IScrollablePaneProps extends React.HTMLAttributes<HTMLElement |
    */
   initialScrollPosition?: number;
 
+  /**
+   * Determines the visibility of vertical and horizontal scrollbars.
+   * If set to ScrollbarVisibility.always, scrollbars are visible always, independent of content overflow.
+   */
   scrollbarVisibility?: ScrollbarVisibility;
+
+  /**
+   * If true, it optimizes the performance which may affect component behavior to some extend in few scenarios.
```

Comment:
This is a very mysterious comment for users; what scenarios are affected?

(Also, "extend" should be spelled "extent"

Reply:
Sticky has stickyClassName prop. The expected visual behavior could be different as the class is applied to both sticky content and to non-sticky content when the component becomes sticky. I'll update the comment.